content/sensu-enterprise/3.7/configuration.md

```diff
@@ -18,10 +18,18 @@ sensu-enterprise service must be restarted before the new values can take effect
 
 ## Sensu Enterprise environment variables
 
-The Sensu Enterprise honors the following environment variables. For
+Sensu Enterprise honors the following environment variables. For
 configuration honored by both Sensu Enterprise and Sensu Core, see the
 [Sensu configuration reference documentation][1].
 
+CMS_OCCUPANCY_FRACTION   | 
+-------------------------|-------
+description              | Tunes the CMSInitiatingOccupancyFraction.
```

Comment:
```suggestion
description              | Configures threshold for the Java CMSInitiatingOccupancyFraction garbage collection strategy.
```

Reply:
Need to add `[2]: https://docs.oracle.com/javase/8/docs/technotes/guides/vm/gctuning/cms.html` at the bottom for this link ref